packages/store/ts/config/v2/table.ts

```diff
@@ -171,8 +176,10 @@ export function resolveTable<input extends TableInput, scope extends Scope = Abi
   input: input,
   scope: scope = AbiTypeScope as unknown as scope,
 ): resolveTable<input, scope> {
-  const namespaceLabel = input.namespaceLabel ?? TABLE_DEFAULTS.namespace;
+  const namespaceLabel = input.namespaceLabel ?? TABLE_DEFAULTS.namespaceLabel;
+  // validate ensures this is length constrained
   const namespace = input.namespace ?? namespaceLabel;
```

Comment:
I find it slightly odd that `namespace` is assigned with `namespaceLabel` rather than vice-versa. This means that `namespaceLabel` is treated as the ID rather than the `namespace`. I'd expect it to be in reverse but I suspect it's due to some constraints I'm not aware of yet.

Reply:
`namespaceLabel` is the "entry point" and the strongly typed thing, which propagates to the namespace if it meets namespace criteria (<=14 chars) of the resource ID, otherwise throws in the validate function and requires either adjusting the label or specifying an explicit `namespace`